migrations/20181204131511-add_canCreateVirtualCards_column_on_collectives.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const canCreateVirtualCardsColName = 'canCreateVirtualCards';
```

Comment:
I don't like the idea of creating a column for that. I will prefer to use `data` or `settings` for that.

Reply:
I wanted this subject to be discussed during the demo, regarding the fact we're planning to make gift cards public should we really keep that feature?

Reply:
Added `canCreateVirtualCards` to settings